pkg/sse/handler.go

```
@@ -78,6 +78,13 @@ func handleSSE(c echo.Context, serviceCtx context.Context, b *Broadcaster, heart
 	heartbeatTicker := time.NewTicker(heartbeatInterval)
 	defer heartbeatTicker.Stop()
 
+	// Try to get the ResponseController to set write deadlines if supported by the underlying ResponseWriter
+	// This helps to detect broken connections more quickly
+	var rc *http.ResponseController
+	if ctrl := http.NewResponseController(c.Response().Writer); ctrl != nil {
```

if ctrl is nil, should we return an error?